README.md

```diff
@@ -93,5 +94,111 @@ When you have any questions, find a bug or have an idea for a feature you can cr
+- **Langfristige Nutzbarkeit**: Halten Sie Ihre Daten über Jahre hinweg organisiert.
+- **Moderationssystem**: Erhalten Sie eine positive, einladende und sichere Umgebung aufrecht.
+- **Notizen**: Schreiben Sie kurze Notizen für sich oder Ihre Klasse.
+- **Kalender**: Behalten Sie den Überblick über Tests und Fristen mit dem wunderschönen Kalender von Dlool.
+- **Stundenplan**: Mit der Stundenplanunterstützung weißt du ganz einfach, welche Fächer wann sind und kannst die Hausaufgaben richtig aufschreiben.
+- **Feiertage**: Schauen Sie nach, wann in Ihrer Gegend Feiertage sind, die Daten werden von [der OpenHolidayAPI](https://www.openholidaysapi.org/en/) bereitgestellt
+  
+- Viele kleine Dinge, die Ihnen das Leben leichter machen.
+
+---
+
+## Technologien
+
+### Frontend
+
+Dlool ist eine progressive Webanwendung und wird von den folgenden Technologien unterstützt:
+
+- [SvelteKit](https://kit.svelte.dev/)
+- [TailwindCSS](https://tailwindcss.com/)
+- [Vercel](https://vercel.com/)
+
+### [Backend](https://github.com/Dlurak/dlool_backend_v2)
+
+Das [Backend](https://github.com/Dlurak/dlool_backend_v2) von Dlool wird von den folgenden Technologien unterstützt:
+
+- [Bun](https://bun.sh)
+- [Elysia](https://elysiajs.com/)
+- [EdgeDB](https://www.edgedb.com/)
+- [Render](https://render.com/)
+
+---
+
+## Erste Schritte
+
+Der einfachste Weg, Dlool auszuprobieren, ist ein Besuch auf [der offiziellen Website](https://dlool.me/).
+
+Wenn Sie Dlool lokal ausführen möchten, z.B. für die Entwicklung, können Sie den Anweisungen unten folgen. ([[#Lokal ausführen]]).
+
+Die Hausaufgaben sind für jeden öffentlich sichtbar, dies ist beabsichtigt, um Mobbing vorzubeugen. Wenn du aber Hausaufgaben erstellen, aktualisieren oder löschen willst, brauchst du einen Account für eine bestimmte Klasse. Du kannst einen Account [hier](https://dlool.me/register) erstellen. Es kann vorkommen, dass jemand aus deiner Klasse bereits Dlool benutzt, dann muss er deine Anfrage, sich für die Klasse anzumelden, akzeptieren. Dies kannst du tun, indem du [diese Seite](https://www.dlool.me/moderation/list) aufrufst und auf die Schaltfläche „Akzeptieren“ klickst.
+
+Sobald Sie Ihr Konto haben, können Sie sich [hier](https://dlool.me/login) anmelden und eine Hausaufgabe [hier](https://dlool.me/homework/) erstellen.
+
+Wenn Sie Fragen haben, einen Fehler finden oder eine Idee für eine Funktion haben, können Sie ein [Issue](https://github.com/Dlurak/dlool_frontend_v2/issues) erstellen.
+
+## Lokal ausführen
+
+1. Klonen Sie dieses Repository
+
+```bash
+   git clone git@github.com:Dlurak/dlool_frontend_v2.git
+   ```
+
+2. Führen Sie das Setup-Skript aus
+
+   ```bash
+   ./setup.sh
+   ```
+
+3. Der Server läuft auf [localhost port 4173](http://localhost:4173)
+
+---
+
+Fühlen Sie sich frei, mich zu kontaktieren, Probleme zu melden oder Ihre Ideen einzubringen, indem Sie ein Issue [hier](https://github.com/Dlurak/dlool_frontend_v2/issues) auf GitHub erstellen.  
```

Comment:
This sentence feels really weird. Also it isn't really contacting *me* but rather to create a issue here on GitHub